### PR TITLE
chore: refactor manage url into a function

### PR DESCRIPTION
**Pull request by lili2311**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Move constructing manageUrl to a more appropriate place.

### CODE REVIEW

File: src/cli/commands/monitor/formatters/format-monitor-response.ts [hunk 1]
```diff
@@ -61,3 +64,17 @@ export function formatMonitorOutput(
...
+function buildManageUrl(resId: string, org?: string): string {
+  const endpoint = url.parse(config.API);
+  let leader = '';
+  if (org) {
+    leader = '/org/' + org;
+  }
+  endpoint.pathname = leader + '/manage';
+  const manageUrl = url.format(endpoint);
+
+  // TODO: what was this meant to do?
```
Comment (lili2311) [inline src/cli/commands/monitor/formatters/format-monitor-response.ts]: @ivanstanev this looks wrong, what is it meant to do?

Comment (dkontorovskyy) [reply src/cli/commands/monitor/formatters/format-monitor-response.ts]: Shall we merge with this, even though line#78 doesn't do anything

Comment (lili2311) [reply src/cli/commands/monitor/formatters/format-monitor-response.ts]: It is the same as it was before, so it should be safe to do so until we can get more answers.

Comment (lili2311) [reply src/cli/commands/monitor/formatters/format-monitor-response.ts]: WDYT?

Comment (dkontorovskyy) [reply src/cli/commands/monitor/formatters/format-monitor-response.ts]: Totally agree. But we need to get some clarification what that was supposed to do and clean it after or refactor

**Pull request merged**